docs/sdk/authentication/session-sigs/get-pkp-session-sigs.md

```diff
@@ -0,0 +1,148 @@
+
+## Example Implementation
+
+The full code implementation can be found [here](https://github.com/LIT-Protocol/developer-guides-code/tree/master/session-signatures/getPkpSessionSigs). 
+
+### Installing the Required Dependencies
+<Tabs
+defaultValue="npm"
+values={[
+{label: 'npm', value: 'npm'},
+{label: 'yarn', value: 'yarn'},
+]}>
+<TabItem value="npm">
+
+```bash
+npm install \
+@lit-protocol/auth-helpers \
+@lit-protocol/constants \
+@lit-protocol/lit-node-client \
+@lit-protocol/contracts-sdk \
+@lit-protocol/lit-auth-client \
+ethers@v5
+```
+
+</TabItem>
+
+<TabItem value="yarn">
+
+```bash
+yarn add \
+@lit-protocol/auth-helpers \
+@lit-protocol/constants \
+@lit-protocol/lit-node-client \
+@lit-protocol/contracts-sdk \
+@lit-protocol/lit-auth-client \
+ethers@v5
+```
+
+</TabItem>
+</Tabs>
+
+The `node-localstorage` dependency is only required when executing code outside a browser environment. The SDK will use the native browser storage when in a browser environment.
+
+### Initializing an Ethers Signer
+The `ETHEREUM_PRIVATE_KEY` environment variable is required.
+```ts
+import { LIT_RPC } from "@lit-protocol/constants";
+import * as ethers from "ethers";
+
+const ethersSigner = new ethers.Wallet(
+  process.env.ETHEREUM_PRIVATE_KEY,
+  new ethers.providers.JsonRpcProvider(LIT_RPC.CHRONICLE_YELLOWSTONE)
+);
+```
+
+### Initializing a `LitNodeClient`
+Here we are initializing an instance of `LitNodeClient` and connecting it to the `datil-test` Lit network.
+```ts
+import { LitNodeClient } from "@lit-protocol/lit-node-client";
+import { LitNetwork } from "@lit-protocol/constants";
+
+let litNodeClient: LitNodeClient;
+
+litNodeClient = new LitNodeClient({
+      litNetwork: LitNetwork.DatilTest,
+      debug: false,
+    });
+await litNodeClient.connect();
+```
+
+### Instantiating a `LitContracts` Instance
+Here we are initializing an instance of `LitContracts`. This allows us to interact with smart contracts on the Lit network. 
+
+```ts
+import { LitContracts } from "@lit-protocol/contracts-sdk";
+import { LitNetwork } from "@lit-protocol/constants";
+
+const litContracts = new LitContracts({
+    signer: ethersSigner,
+    network: LitNetwork.DatilTest,
+    debug: false,
+});
+await litContracts.connect();
+```
+
+### Generating Session Signatures
+In this example, we're granting the capability to request to decrypt any data that we may be authorized to decrypt (i.e. we satisfy the Access Control Conditions the data was encrypted with). We could, however, specify the [LitAccessControlConditionResource](https://v6-api-doc-lit-js-sdk.vercel.app/classes/auth_helpers_src.LitAccessControlConditionResource.html) for specific encrypted data we're permitting the decryption capability for. In real-world applications, it's more common and secure to limit access to specific Lit resources instead of specifying the wildcard (`"*"`) identifier.
```

Comment:
same comment as above regarding decryption capabilities vs pkp signing

Reply:
Similar to the Lit Actions change:

> In this example, we're enabling our session to use a PKP for signing.
> 
> The current code uses the wildcard (`*`) identifier for `LitPKPResource`, which grants signing abilities to **any** PKP. This should only be used for example implementations or debugging. A more secure implementation would instead use the PKP `tokenId` to grant signing abilities to a specific PKP.